worker/util.go

```diff
@@ -100,3 +113,10 @@ func detectResources(conf config.Resources) config.Resources {
 // that doesn't do anything.
 func NoopTaskRunner(l TaskControl, c config.Worker, j *pbf.TaskWrapper) {
 }
+
+const (
+	b  = 1
+	kb = 1024 * b
+	mb = 1024 * kb
+	gb = 1024 * mb
```

Comment:
Oh, I was forgetting the context. The unit should match whatever gopsutil is returning. Also, "b", "kb", and "mb" aren't used anywhere, right?